content/applications/inventory_and_mrp/inventory/management/lots_serial_numbers/serial_numbers.rst

```diff
@@ -1,147 +1,225 @@
-specify its serial number. Of course, you have several ways of doing so:
-
--  Manual assignation
-
--  Multi-assignation
-
--  Copy/pasting from an Excel file
-
-Whatever could be your favorite way to assign serial numbers, you
-firstly need to open the detailed operations of your picking.
-
-.. image:: serial_numbers/serial_numbers_04.png
-    :align: center
-
-Manual assignation of the different serial numbers
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-When opening the detailed operations, you can click on *Add a line*.
-Then, you will be able to fill in the serial number field. Once done,
-you can click again on *Add a line* to register a new serial number.
-
-.. image:: serial_numbers/serial_numbers_05.png
-    :align: center
-
-Multi-assignation
-~~~~~~~~~~~~~~~~~
-
-By using the multi-assignation of serial numbers, Odoo will
-automatically create the necessary lines. To do so, you have to enter
-the first serial number of your set and the number of products you have
-to assign a serial number to.
-
-Once done, click on *Assign Serial Numbers* and Odoo will do the rest.
-
-.. image:: serial_numbers/serial_numbers_06.png
-    :align: center
-
-.. image:: serial_numbers/serial_numbers_07.png
-    :align: center
-
-Copy/pasting from an Excel file
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-To use copy/pasting, open the spreadsheet containing the serial numbers
-you received and copy the list. Then, past them in the *Lot/Serial
-Number Name* column of the wizard. Doing so, Odoo will automatically
-create the lines you need.
-
-.. image:: serial_numbers/serial_numbers_08.png
-    :align: center
-
-.. image:: serial_numbers/serial_numbers_09.png
-    :align: center
-
-.. image:: serial_numbers/serial_numbers_10.png
-    :align: center
-
-Operation types
-===============
-
-You also have the possibility to define how you will manage lots for
-each operation type. To define it, go to :menuselection:`Configuration --> Operation Types` in the
-*Inventory* app.
-
-For each type, you can decide if you allow the creation of new lot
-numbers or want to use existing ones. By default, the creation of new
-lots is only allowed at product reception.
-
-.. image:: serial_numbers/serial_numbers_11.png
-    :align: center
+New serial numbers can be created for products already in stock with no assigned serial number. To
+do this, go to :menuselection:`Inventory --> Products --> Lots/Serial Numbers`, and click
+:guilabel:`Create`. A new :guilabel:`Lot/Serial Number` will be generated automatically.
 
 .. tip::
-      If you have inter-warehouse transfers and track products by lots, it can
-      be useful to allow using existing lot numbers in receipts too.
+    Although Odoo automatically generates a new :guilabel:`Lot/Serial Number` to follow the most
+    recent number, it can be edited and changed to be whatever number is desired by clicking the
+    line under the :guilabel:`Lot/Serial Number` field and changing the generated number.
```

Comment:
Rephrase and fix indentation
```suggestion
   Although Odoo automatically generates a new serial number to follow the most
   recent number, it can be changed to whatever number is desired by clicking the
   line under the :guilabel:`Lot/Serial Number` field and editing the generated number.
```